src/views/StatsView.vue

```diff
@@ -60,16 +60,20 @@
           with play-by-play commentary.
         </p>
       </div>
+
       <!-- Rankings Table -->
-      <template
-        v-if="selectedSeason && selectedSeason.rankings && selectedSeason.rankings.length > 0"
-      >
+      <template v-if="weeklyRanking">
         <h2 class="text-h2 mb-4">
           Weekly Rankings
           <stats-scoring-dialog />
         </h2>
         <stats-leaderboard :loading="loadingData" :season="selectedSeason" />
       </template>
+      <!-- Error display -->
+      <div v-if="error" class="d-flex flex-column align-center text-center">
+        <h2>Oops! There was a problem loading the leaderboard. Refresh the page to try again</h2>
```

Comment:
Let's make this a <p>
```suggestion
        <p class="text-body-1">Oops! There was a problem loading the leaderboard. Refresh the page to try again</p>
```